web/client/components/map/cesium/Map.jsx

```diff
@@ -173,7 +173,7 @@ class CesiumMap extends React.Component {
             let {type, ...tpOptions} = rawOptions.terrainProvider;
             switch (type) {
             case "cesium": {
-                overrides.terrainProvider = new Cesium.CesiumTerrainProvider(tpOptions);
```

Comment:
I think this should stay as is is if it can work after the API changes, with new options
The case entry, otherwise, removed. 
If a terrainProvider must be added, you should instead change the default terrainProvider option to "ellipsoid" (in viewer and localConfig, and add an entry in this switch case for "ellipsoid", that is also the switch case "default". 
doing this, also if we do not support cesium anymore, the switch will fall back to ellipsoid.

Reply:
Sorry, I didn't see the comment in the related issue. Please change the defaults and add a default/ellipsoid entry to this switch. It is a cleaner solution.